Source/MOLAuthenticatingURLSession.m

```diff
@@ -60,33 +60,28 @@ - (void)setUserAgent:(NSString *)userAgent {
   self.sessionConfig.HTTPAdditionalHeaders = addlHeaders;
 }
 
-#pragma mark Server Roots properties
+#pragma mark Server Roots
 
 - (void)setServerRootsPemFile:(NSString *)serverRootsPemFile {
-  if (!serverRootsPemFile) {
-    _serverRootsPemFile = nil;
-    return;
-  }
+  if (!serverRootsPemFile) return;
```

Comment:
This means passing nil won't reset back to the system roots. This line should probably be:

if (!serverRootsPemFile) return [self setServerRootsPemData:nil];